### PR TITLE
Persian Translations

### PR DESCRIPTION
**Pull request by alirezaahani**
Translation for Persian.
I couldn't find any word that meant " shader pack " in Persian and still made sense to most people so I didn't translate it .
Also names like "Iris" never get translated.

### CODE REVIEW

File: src/main/resources/assets/iris/lang/fa_ir.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "iris.keybind.reload": "بارگیری مجدد shader ها",
+  "iris.keybinds": "Iris",
+  "iris.shaders.reloaded.failure": "خطا در بارگیری مجدد shader ها ، علت : %s",
+
+  "options.iris.apply": "اعمال",
+  "options.iris.refresh": "تازه سازی",
+  "options.iris.openShaderPackFolder": "باز کردن پوشه shader pack ها...",
+  "options.iris.refreshShaderPacks": "تازه سازی shader pack",
+  "options.iris.shaderPackSelection": "Shader Packs...",
+  "options.iris.shaderPackSelection.title": "Shader Packs",
```
Comment (maxigator) [inline src/main/resources/assets/iris/lang/fa_ir.json:12]: If you don't change the translation you can remove these lines too

File: src/main/resources/assets/iris/lang/fa_ir.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "iris.shaders.reloaded": "shader ها بارگیری مجدد شدند!",
+  "iris.keybind.reload": "بارگیری مجدد shader ها",
+  "iris.keybinds": "Iris",
```
Comment (maxigator) [inline src/main/resources/assets/iris/lang/fa_ir.json:4]: If you don't change the translation you can remove this line

**Review by maxigator — CHANGES_REQUESTED** [review]

**Review by alirezaahani — COMMENTED** [review]
Thanks.

**Review by coderbot16 — APPROVED** [review]
Thanks!

**Pull request merged**